server.js

```diff
@@ -2,28 +2,72 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import Thought from './models/thought'
+import thoughtsData from './data/thoughts.json'
+
+const ERR_COULD_NOT_SAVE_THOUGHT = 'Could not save thought to database'
+
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+
+// Created test data to get the database deployed in Mongo Atlas
+if (process.env.RESET_DATABASE) {
+  console.log('Resetting database ...')
+
+  const seedDatabase = async () => {
+    await Thought.deleteMany()
+    await thoughtsData.forEach((thought) => new Thought(thought).save())
+  }
+  seedDatabase()
+}
+
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
-// Start the server
+// Get thoughts, sort by createdAt descending order, limit to show 20
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thoughts)
+})
+
+// Retreive the information sent by the client to API endpoint, don't include hearts
+app.post('/thoughts', async (req, res) => {
```

Comment:
Although, we _can_ have a get and a post that are named the same, so change this to `'/'` as well